spec/plugin_mixin/http_client_ssl_spec.rb

```diff
@@ -44,6 +44,18 @@
   end
 end
 
+shared_examples 'an obsolete setting with guidance' do |deprecations_and_guidance|
+
+  let(:logger_stub) { double('Logger').as_null_object }
```

Comment:
I'm not seeing how this is used? In other examples we have a `before` block that stubs `:logger` explicitly with this but not in this example. Maybe i'm missing some inheritance?

Reply:
It's not - cut and paste error 🤦‍♂️